src/code-templates/libraries/jwt-token-verifier/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "JWT for Practica",
   "main": ".dist/lib/index.js",
   "scripts": {
+    "installDependencies": "npm install",
```

Comment:
@rluvaton This is actually not needed in any other package - Turbo will install automatically. But it's a good idea to put it here for the sake of uniformity